src/routes/console/project-[project]/functions/function-[function]/rebuild.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+    import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import { Modal } from '$lib/components';
+    import { Button } from '$lib/elements/forms';
+    import { addNotification } from '$lib/stores/notifications';
+    import { sdk } from '$lib/stores/sdk';
+    import type { Models } from '@appwrite.io/console';
+    import { createEventDispatcher } from 'svelte';
+
+    export let showRebuild = false;
+    export let selectedDeployment: Models.Deployment = null;
+
+    const dispatch = createEventDispatcher();
+
+    const handleSubmit = async () => {
+        try {
+            await sdk.forProject.functions.createBuild(
+                selectedDeployment.resourceId,
+                selectedDeployment.$id,
+                selectedDeployment.buildId
+            );
+            showRebuild = false;
+            addNotification({
+                type: 'success',
+                message: `Retrying build`
+            });
+            dispatch('rebuild');
+            trackEvent(Submit.DeploymentUpdate);
+        } catch (error) {
+            addNotification({
+                type: 'error',
+                message: error.message
+            });
+            trackError(error, Submit.DeploymentUpdate);
+        }
+    };
+</script>
+
+<Modal bind:show={showRebuild} onSubmit={handleSubmit}>
+    <svelte:fragment slot="header">Retry build Deployment</svelte:fragment>
+    <p>Are you sure you want to retry build this deployment?</p>
```

Comment:
Not quite right. Please update to:

```suggestion
    <svelte:fragment slot="header">Retry build</svelte:fragment>
    <p>Are you sure you want to retry building this deployment?</p>
```